src/fstab-generator/fstab-generator.c

```diff
@@ -856,6 +887,10 @@ static int add_sysroot_mount(void) {
                 /* This is handled by gpt-auto-generator */
                 log_debug("Skipping root directory handling, as gpt-auto was requested.");
                 return 0;
+        } else if (streq(arg_root_what, "fstab")) {
+                /* This is handled by parse_fstab */
+                log_debug("Using fstab for root directory handling as requested.");
```

Comment:
Maybe "initrd's fstab" for clarity?
"Using initrd's fstab for /sysroot/ configuration" ?